Modules/Microsoft365DSC/Modules/M365DSCReport.psm1

```diff
@@ -598,6 +699,31 @@ function New-M365DSCReportFromConfiguration
         [System.String]
         $OutputPath
     )
+    DynamicParam # parameter 'Delimiter' is only available when Type = 'CSV'
+    {
+        $paramDictionary = [System.Management.Automation.RuntimeDefinedParameterDictionary]::new()
+        if ($Type -eq 'CSV')
+        {
+            $delimiterAttr = [System.Management.Automation.ParameterAttribute]::New()
+            $delimiterAttr.Mandatory = $false
+            $attributeCollection = [System.Collections.ObjectModel.Collection[System.Attribute]]::New()
+            $attributeCollection.Add($delimiterAttr)
+            $delimiterParam = [System.Management.Automation.RuntimeDefinedParameter]::New("Delimiter", [System.String], $attributeCollection)
+            $delimiterParam.Value = ';' # default value, comma makes a mess when importing a CSV-file in Excel
+            $paramDictionary.Add("Delimiter", $delimiterParam)
+        }
+        return $paramDictionary
+    }
+
+begin
+{
+    if ($PSBoundParameters.ContainsKey('Delimiter'))
+    {
+        $Delimiter = $PSBoundParameters.Delimiter
+    }
+}
+process # required with DynamicParam
+{
```

Comment:
Indentation is not correct, leading to a curly bracket on the same line as another end block on line 778.